api/views/phones.py

```diff
@@ -582,7 +601,29 @@ def inbound_sms_iq(request: Request) -> response.Response:
     relay_number, real_phone = _get_phone_objects(relay_num)
     _check_remaining(relay_number, "texts")
 
-    # TODO: handle replies
+    if from_num == real_phone.number:
+        try:
```

Comment:
Tried to test replies and reply when text message limit is hit but I instead got the following error from iQ:
```
{"detail":"Submitting message has failed","reason":"STATUS_SUBMITFAIL","result":{"fragmentCount":1,"from":"14135005717","messageType":"SMS","referenceId":"","resultResponses":[{"to":"16467613411","status":"FAILURE"}],"text":"debugging"},"success":false}
```

Reply:
Sounds like it was a config on iQ side. You should be able to test again now.